.circleci/config.yml

```diff
@@ -1,5 +1,34 @@
+version: 2.1
+
 jobs:
-  build:
+  single_node_tests:
+    docker:
+      - image: cimg/base:2020.01
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install debugging utils
+          command: |
+            sudo apt-get update -y
```

Comment:
is this really needed? seems like just a waste of cpu and testing time

Reply:
not really, so I just removed that block